examples/generative/md/text_generation_with_miniature_gpt.md

```diff
@@ -326,236 +349,1261 @@ model.fit(text_ds, verbose=2, epochs=25, callbacks=[text_gen_callback])
 <div class="k-default-codeblock">
 ```
 Epoch 1/25
-391/391 - 135s - loss: 5.5949 - dense_2_loss: 5.5949
+
+WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
+I0000 00:00:1699058504.861781   62899 device_compiler.h:187] Compiled cluster using XLA!  This line is logged at most once for the lifetime of the process.
+
+ 1/1 ━━━━━━━━━━━━━━━━━━━━ 1s 617ms/step
+ 1/1 ━━━━━━━━━━━━━━━━━━━━ 0s 16ms/step
```

Comment:
The repeated progress bars look like a bug under `verbosity=2`. Looking into it.